components/contactPicker/index.html

```diff
@@ -14,6 +14,7 @@
                     <span class="fa fa-check"></span>
                 </label>
                 <span>{{ person.displayName }}</span>
+                <span class="phone-number">{{ person.phoneNumbers[0].value }}</span>
```

Comment:
I believe you can have a contact without any phone numbers in which case this would throw an `undefined` error.